implementations/contracts/ERC725.sol

```diff
@@ -28,22 +21,16 @@ contract ERC725 is ERC725XCore, ERC725YCore {
      * @custom:requirements
      * - `initialOwner` CANNOT be the zero address.
      */
-    constructor(address initialOwner) payable {
-        if (initialOwner == address(0)) {
-            revert OwnableCannotSetZeroAddressAsOwner();
-        }
-        OwnableUnset._setOwner(initialOwner);
-    }
+    constructor(
+        address initialOwner
+    ) payable ERC725X(initialOwner) ERC725Y(initialOwner) {}
 
     /**
-     * @inheritdoc ERC725XCore
+     * @inheritdoc ERC725X
      */
     function supportsInterface(
         bytes4 interfaceId
-    ) public view virtual override(ERC725XCore, ERC725YCore) returns (bool) {
-        return
-            interfaceId == _INTERFACEID_ERC725X ||
-            interfaceId == _INTERFACEID_ERC725Y ||
-            super.supportsInterface(interfaceId);
+    ) public view virtual override(ERC725X, ERC725Y) returns (bool) {
+        return super.supportsInterface(interfaceId);
```

Comment:
Does this return `true` for both ERC725X & ERC725Y interface ids?
```suggestion
        return (
                ERC725X.supportsInterface(interfaceId) ||
                ERC725Y.supportsInterface(interfaceId) ||
                super.supportsInterface(interfaceId)
        );
```

Reply:
@b00ste yes it does, `super` will call the `supportsInterface(...)` function in each parent contract in the order of inheritance

```
ERC725.supportsInterface(bytes4)
|
v
super.supportsInterface(bytes4) <--- ERC725X.supportsInterface(bytes4)
                                                                 |
                                                                 v
                                                                 super.supportsInterface(bytes4) <---- ERC725Y.supportsInterface(bytes4)
                                                                                                                                    |
                                                                                                                                    v
                                                                                                                                    super.supportsInterface(bytes4) <--- ERC165..supportsInterface(bytes4)
                                                      
```

Reply:
Double checked and tested in this commit: https://github.com/ERC725Alliance/ERC725/pull/253/commits/4acb5843cb9a4d5b29aebe8ff4186116e922d8c5